src/main/java/dataInt/Ejercicio1.java

```diff
@@ -0,0 +1,34 @@
+package dataInt;
+
+import com.github.javafaker.Faker;
+
+import java.util.Arrays;
+
+public class Ejercicio1 {
+    public static void main(String[] args) {
+        final var listaArray = new int[50];
+        for (int i = 0; i <= listaArray.length - 1; i++) {
+            listaArray[i] = new Faker().number().numberBetween(1, 1000);
+        }
+        valoresAleatorias(listaArray);
+        resultadoMultiplicado(nuevoArray(listaArray));
+    }
+
+    //Metodo que Multiplica los valores Aleatorios
+    public static int[] nuevoArray(int[] nuevaListaArray) {
+        for (int i = 0; i <= nuevaListaArray.length - 1; i++) {
```

Comment:
final var n = listaNumeros.length;
for(var i = 0; i < n; i++)

Reply:
Se corrigio!